bin/pycbc_make_skymap

```diff
@@ -155,7 +159,8 @@ def main(trig_time, mass1, mass2, spin1z, spin2z, f_low, f_upper, sample_rate,
         "--autogating-pad","16",
         "--strain-high-pass", str(highpass_frequency),
         "--pad-data", "8",
-        "--chisq-bins",'0.72*get_freq("fSEOBNRv4Peak",params.mass1,params.mass2,params.spin1z,params.spin2z)**0.7']
+        "--chisq-bins",'0.72*get_freq("fSEOBNRv4Peak",params.mass1,params.mass2,params.spin1z,params.spin2z)**0.7',
+        "--injection-file", str(opt.injection_file)]
```

Comment:
What happens here if I do *not* want to use an injection file?

Reply:
Wow, okay, I didn't update my code locally correctly. I will do some more testing and get back to you.

Reply:
I've used `if opt.injection_file is not None`, there must be a more elegant way of writing this?